src/graph_ir/tests/test_low_graph_ir.cc

```diff
@@ -0,0 +1,19 @@
+#include "low_graph_ir.h"
```

Comment:
You have these files that have `test` in the title, but they don't seem to be hooked up to any test rules. Please do so so they can be automatically run.

Reply:
Do we have cpp test rules yet? I thought we only had datalog test rules.

I also don't know how to do this. Can I do it in a follow-up PR so that I can get this in and then look it up after ?

Reply:
I just imported gtest into our repo. See https://github.com/google-research/raksha/blob/7ed3da8347f513249c8508f63ba265640ad1f063/src/analysis/souffle/cpp_interface_test.cc#L25 for an example.